core/src/edu/cornell/gdiac/main/BouncePlatformModel.java

```diff
@@ -19,11 +19,14 @@ public class BouncePlatformModel extends PlatformModel {
 
   private float defaultMaxSpeed;
 
+  private int offsetX;
```

Comment:
Why do we need to offset the bounce platforms? This is kind of a rough solution, can you provide a justification as to why there's not a better way?